pkg/webhook/controlplane/ensurer.go

```diff
@@ -200,38 +201,41 @@ var (
 			EmptyDir: &corev1.EmptyDirVolumeSource{},
 		},
 	}
-	konnectivityUdsVolumeMount = corev1.VolumeMount{
-		Name:      "konnectivity-uds",
-		MountPath: "/konnectivity-uds",
-		ReadOnly:  false,
-	}
-	konnectivityUdsEnvVar = corev1.EnvVar{
-		Name:  "AUDIT_KONNECTIVITY_UDS_SOCKET",
-		Value: "/konnectivity-uds/konnectivity-server.socket",
-	}
 	konnectivityMtlsVolumeMounts = []corev1.VolumeMount{
 		{
 			Name:      "ca",
-			MountPath: "/konnectivity-proxy/ca",
+			MountPath: "/proxy/ca",
 			ReadOnly:  true,
 		},
 		{
 			Name:      "konnectivity-server-client-tls",
-			MountPath: "/konnectivity-proxy/client",
+			MountPath: "/proxy/client",
 			ReadOnly:  true,
 		},
 	}
-	konnectivityMtlsEnvVars = []corev1.EnvVar{
```

Comment:
Does this mean the support for konnectivity is dropped when rolling this out?

Reply:
No; konnectivity with mTLS is still supported; only the UDS proxy gets dropped.

The proxy environment variable is set in the ensureAuditForwarderProxy function, with the proxy host depending on which kind of proxy it is; therefore the global variable is not necessary any more.